<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNewProjectDialog</class>
 <widget class="QDialog" name="TNewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0" columnminimumwidth="0,0,0,0">
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxPanelTypes">
       <item>
        <property name="text">
         <string>VARIA-SL50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VARIA-SL80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VARIA-80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VARIA-100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VARIA-150</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MT-702</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MD-702</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MT-1002</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MD-1002</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MT-2002</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXT-701</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXD-701</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXT-1001</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXR-1001</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXD-1001</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXT-1901-PAN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXD-1901-PAN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXT-2001-PAN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MXD-2001-PAN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TheoSys</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBoxResolutions"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelResolution">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditJobName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelJobName">
       <property name="text">
        <string>Job name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPanelType">
       <property name="text">
        <string>Panel type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLabel" name="labelPanelImage">
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxGenFiles">
       <property name="text">
        <string>Use system generated file name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Description</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QLabel" name="labelDescription">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonPrevious">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFinish">
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
